net/autossh/src/opnsense/mvc/app/controllers/VerbNetworks/Autossh/Api/ServiceController.php

```diff
@@ -0,0 +1,228 @@
+<?php
+
+/*
+    Copyright (c) 2018 Verb Networks Pty Ltd <contact@verbnetworks.com>
+    Copyright (c) 2018 Nicholas de Jong <me@nicholasdejong.com>
+    All rights reserved.
+
+    Redistribution and use in source and binary forms, with or without modification,
+    are permitted provided that the following conditions are met:
+
+    1. Redistributions of source code must retain the above copyright notice, this
+       list of conditions and the following disclaimer.
+
+    2. Redistributions in binary form must reproduce the above copyright notice,
+       this list of conditions and the following disclaimer in the documentation
+       and/or other materials provided with the distribution.
+
+    THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND
+    ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED
+    WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+    DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE FOR
+    ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES
+    (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES;
+    LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON
+    ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+    (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
+    SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+*/
+
+namespace VerbNetworks\Autossh\Api;
+
+use \OPNsense\Base\ApiControllerBase;
+use \OPNsense\Core\Backend;
+use \OPNsense\Base\UIModelGrid;
+use \VerbNetworks\Autossh\Autossh;
+
+class ServiceController extends ApiControllerBase
+{
+
+    public function isConfigChangeAction()
+    {
+        $response = array("status"=>"fail", "message" => "Invalid request");
+
+        if ($this->request->isGet()) {
+            $model = new Autossh();
+            $response = array("status"=>"success", "data" => $model->isConfigChange(), "message"=>null);
+        }
+        
+        return $response;
```

Comment:
The $response variable is not needed

Reply:
Returning early has been applied in all controllers - it's an old habit